rapyuta_io_sdk_v2/config.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# Copyright 2024 Rapyuta Robotics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+from dataclasses import dataclass
+
+from rapyuta_io_sdk_v2.constants import (
+    NAMED_ENVIRONMENTS,
+    PROD_ENVIRONMENT_SUBDOMAIN,
+    STAGING_ENVIRONMENT_SUBDOMAIN,
+)
+from rapyuta_io_sdk_v2.exceptions import ValidationError
+
+
+@dataclass
```

Comment:
todo: You can remove this since we are not really using this as a dataclass.

Give this a read if you still want to use it: https://realpython.com/python-data-classes/

You can modify the class accordingly.